make/photon/registry/Dockerfile.base

```diff
@@ -3,4 +3,5 @@ FROM photon:5.0
 RUN tdnf install -y shadow >> /dev/null \
     && tdnf clean all \
     && mkdir -p /etc/registry \
-    && groupadd -r -g 10000 harbor && useradd --no-log-init -m -g 10000 -u 10000 harbor
+    && groupadd -r -g 10000 harbor && useradd --no-log-init -m -g 10000 -u 10000 harbor \
+    && chown -R harbor:harbor /etc/pki/tls/certs
```

Comment:
why moves chown -R harbor:harbor /etc/pki/tls/certs into Dockerfile.base?

Reply:
The files under `/etc/pki/tls/certs` are not affected by binary builds (or any changes to the harbor sources).
They are only affected by the base Photon image.
Therefore, it is not necessary to chown the registry-photon image every time it is created.

Reply:
this is the dockerfile for base image, it will no be built every time but on demand to refresh the OS libraries. 

I prefer to put all the permission handling together, it's easy to check and maintain. Otherwise, we have to know that there is a separate one in the base image.

Reply:
> this is the dockerfile for base image, it will no be built every time but on demand to refresh the OS libraries.

I understand that.

If `chown -R harbor:harbor /etc/pki/tls/certs` is in the `Dockerfile` instead of the `Dockerfile.base` (before this fix), it will be executed on every image build even if the OS libraries are not changed. This is obviously a wasteful process.
With this fix, `chown -R harbor:harbor /etc/pki/tls/certs` will be executed only if there are changes in OS libraries.

However, it's not a significant process, and I'm not particularly particular about it, so I'll revert it.

Reply:
Pushed.